yoonsang/Week 2/acmicpc28702.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+  char a[3][9];
+  int b,c;
+  scanf("%s %s %s", a[0], a[1], a[2]);
+  for(int i=0;i<3;i++){
+    b=atoi(a[i]);
+    if(b==0) continue;
+    else{ c=b+3-i; break; }
+  }
+  if(c%3==0){
+    if(c%5==0) { printf("FizzBuzz"); return 0; }
+    printf("Fizz"); return 0;
+  }
+  if(c%5==0) { printf("Buzz"); return 0; }
+  printf("%d", c);
+}
```

Comment:
혹시 가시성이 좋게 하는 게 목적이시라면 
```suggestion
 if (c % 15 == 0) {  // FizzBuzz는 3과 5의 공배수이므로 15로 처리
        printf("FizzBuzz");
    } else if (c % 3 == 0) {
        printf("Fizz");
    } else if (c % 5 == 0) {
        printf("Buzz");
    } else {
        printf("%d", c);
    }
}
```

이 정도가 괜찮을 것 같습니다.